chunks/lang-node/Dockerfile

```
@@ -9,12 +9,15 @@ USER gitpod
 ENV TRIGGER_REBUILD=1
 ENV NODE_VERSION=${NODE_VERSION}
 
+ENV PNPM_HOME=/home/gitpod/.pnpm
+ENV PATH=/home/gitpod/.nvm/versions/node/v${NODE_VERSION}/bin:/home/gitpod/.yarn/bin:${PNPM_HOME}:$PATH
+
 RUN curl -fsSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | PROFILE=/dev/null bash \
     && bash -c ". .nvm/nvm.sh \
         && nvm install v${NODE_VERSION} \
         && nvm alias default v${NODE_VERSION} \
-        && npm install -g typescript yarn pnpm node-gyp" \
+        && npm install -g yarn \
+        && yarn global add typescript yarn pnpm node-gyp" \
```

@gengjiawen let's use the original code to install global dependencies for now:

```suggestion
        && npm install -g typescript yarn pnpm node-gyp" \
```

This way, users depending on `nvm` to switch between nodejs versions do not have broken workflows, where maybe they were expecting to have the ability to have different versions of tooling with different nodejs versions. In other words, global dependencies should be for a nodejs version, not the platform as a whole.

I recommend you maintain a custom image if you wish to use yarn to avoid having to reinstall global dependencies when changing nodejs versions with nvm. Also, if you'd like to socialize a change (for the future) to use yarn (like you did here), may I ask you to create an issue so we can socialize? This way, we can avoid delay this PR (which nicely fixes two issues already).